Cask

```diff
@@ -1,4 +1,6 @@
 (source "melpa" "https://melpa.org/packages/")
 
 (development
- (depends-on "cl-lib"))
+ (depends-on "cl-lib")
+ (depends-on "ecukes")
```

Comment:
the commit seems to add dependencies, not remove them.

Reply:
That's weird :astonished:

Reply:
This was some history manipulation gone wrong. Looks like I tried to move the removal from the preceding commit to this separate commit but ended up not removing it from the other commit and instead reverting it in the new commit. Fixed.